### ISSUE TITLE
Cannot access RandomFourierFeatures keras layer in TensorFlow 2.0

### ISSUE THREAD

**Issue by jejjohnson** [tensorflow/tensorflow#33140]
### Problem Summary

Hello,

I'm not entirely sure if this is a bug or if I have made an error in installation but I have not been able to access the `tf.keras.layers.kernelized.RandomFourierFeatures` module within the tensorflow 2.0 framework.

### Problem Description

I've seen the `kernelized` and `RandomFourierFeatures` layer within the code-base within the [init file](https://github.com/tensorflow/tensorflow/blob/e62dc433fcce833313b4174e20fc24c418593d27/tensorflow/python/keras/layers/__init__.py), the [actual layer](https://github.com/tensorflow/tensorflow/blob/ddde6c74155df8da7229854d5387f3df64ead88a/tensorflow/python/keras/layers/kernelized.py) and the [test file](https://github.com/tensorflow/tensorflow/blob/b4245c36bc9bcb752c0a2118728098b359fd97e2/tensorflow/python/keras/layers/kernelized_test.py) under the [keras layers](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/keras/layers) directory. However, I cannot call the layer from anywhere within the tensorflow layers frameworks.

**Current Behaviour for importing `Kernelized` layer**

<details>

```python
tf.keras.layers.kernelized
```

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-122e54d3d99d> in <module>()
----> 1 tf.keras.layers.kernelized

AttributeError: module 'tensorflow_core.keras.layers' has no attribute 'kernelized'
```
</details>

**Current Behaviour for importing `RandomFourierFeatures` layer**

<details>

```python
tf.keras.layers.RandomFourierFeatures
```

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-058ec5bf6f57> in <module>()
----> 1 tf.keras.layers.RandomFourierFeatures

AttributeError: module 'tensorflow_core.keras.layers' has no attribute 'RandomFourierFeatures'
```

</details>

---
### System Information

I am currently using a Google Colab Notebook. [Link to Colab Notebook](https://colab.research.google.com/drive/1lMnpHXnP5BTAWy8vyKtEYgmLbq2Dlw3G).

**TensorFlow Version**

<details>

```python
import tensorflow as tf
tf.__version__
!python --version
```

```python
'2.1.0-dev20191008'
Python 3.6.8
```
</details>

**Colab CPU Info**

<details>

```python
processor	: 0
vendor_id	: GenuineIntel
cpu family	: 6
model		: 63
model name	: Intel(R) Xeon(R) CPU @ 2.30GHz
stepping	: 0
microcode	: 0x1
cpu MHz		: 2300.000
cache size	: 46080 KB
physical id	: 0
siblings	: 2
core id		: 0
cpu cores	: 1
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 13
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm invpcid_single ssbd ibrs ibpb stibp fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt arat md_clear arch_capabilities
bugs		: cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs
bogomips	: 4600.00
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:

processor	: 1
vendor_id	: GenuineIntel
cpu family	: 6
model		: 63
model name	: Intel(R) Xeon(R) CPU @ 2.30GHz
stepping	: 0
microcode	: 0x1
cpu MHz		: 2300.000
cache size	: 46080 KB
physical id	: 0
siblings	: 2
core id		: 0
cpu cores	: 1
apicid		: 1
initial apicid	: 1
fpu		: yes
fpu_exception	: yes
cpuid level	: 13
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm invpcid_single ssbd ibrs ibpb stibp fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt arat md_clear arch_capabilities
bugs		: cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs
bogomips	: 4600.00
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:
```

</details>

**Memory Info**

<details>

```python
MemTotal:       13341992 kB
MemFree:         8670124 kB
MemAvailable:   12753184 kB
Buffers:          115848 kB
Cached:          3865368 kB
SwapCached:            0 kB
Active:          1160124 kB
Inactive:        3077576 kB
Active(anon):     256804 kB
Inactive(anon):      324 kB
Active(file):     903320 kB
Inactive(file):  3077252 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:             0 kB
SwapFree:              0 kB
Dirty:               772 kB
Writeback:             0 kB
AnonPages:        256548 kB
Mapped:           167332 kB
Shmem:               656 kB
Slab:             377108 kB
SReclaimable:     355904 kB
SUnreclaim:        21204 kB
KernelStack:        3040 kB
PageTables:         4044 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:     6670996 kB
Committed_AS:    1194088 kB
VmallocTotal:   34359738367 kB
VmallocUsed:           0 kB
VmallocChunk:          0 kB
AnonHugePages:         0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
DirectMap4k:       59380 kB
DirectMap2M:     4134912 kB
DirectMap1G:    11534336 kB
```
</details>

Thank you.

**Comment by gowthamkpr** [comment]
I have made a few changes in the colab notebook and its working fine for me. 
In tensorflow 2.0 you can access RandomFourierFeatures Keras layers this way

`from tensorflow.python.keras.layers.kernelized import RandomFourierFeatures`

Please find the github gist [here](https://colab.sandbox.google.com/gist/gowthamkpr/0ea14d917d5f8f841570cc92b7ef69fb/copy-of-rff_layer.ipynb).

**Comment by jejjohnson** [comment]
Thanks for looking into it.

I noticed that the tensorflow version in the gist says that it is tensorflow version `1.15.0-rc3`.

Does this mean that it is not available for version `2.0.0` or `2.1.0-dev20191008`?

Thanks again.
